docs/usage-cli.rst

```diff
@@ -68,3 +68,94 @@ If you’re using ``zsh``, add the following to your ``.zshrc``:
   autoload bashcompinit
   bashcompinit
   eval "$(register-python-argcomplete airflow)"
+
+Exporting DAGs to images
+------------------------
+
+The application has the functionality of saving DAG to image file. You can attach them to the documentation
+or share with other people without having to send the DAG file and need to install the application.
+However, you need to have `Graphviz <https://graphviz.gitlab.io/download/>`_ installed to generate it.
+
+For example, if you want to export ``example_gcp_datacatalog`` DAG then you can use the following command:
+
+.. code-block:: bash
+
+  airflow dag show example_gcp_datacatalog
```

Comment:
Should we use more generic example? Not all users have to have GCP installed. Additionally, where should be the DAG file? Does this work only for DAGs in `AIRFLOW__CORE__DAGS_FOLDER`?

Reply:
I will add a description of the `subdir` switch.  I wanted to show that CLI can also show very complex DAGs. Then this features is the most useful.  Unfortunately, other examples are relatively simple. This is just an example. so it can refer to specific operators.

Reply:
I agree with @nuclearpinguin , we should add 1 example for our tutorial DAG or any other DAGs from https://github.com/apache/airflow/tree/master/airflow/example_dags